### PR TITLE
chore: Added Slackware 15.0 test case via identify_os_release

### PR DESCRIPTION
**Pull request by mco-system**
Removed identify_slackware() from  xe-linux-distribution
Added call to identify_os_release() in xe-linux-distribution-test
Added Slackware 15.0 test case via identify_os_release()

Signed-off-by: mco-system <michael.costa@mcos.nc>

### CODE REVIEW

File: mk/xe-linux-distribution [hunk 1]
```diff
@@ -135,10 +135,10 @@ identify_redhat()
     # Enterprise Linux Enterprise Linux Server release 5 (Carthage)
     # Enterprise Linux Enterprise Linux Server release 5.5 (Carthage)
     # Oracle Linux Server release 5.6
-    
+
```
Comment (stormi) [inline mk/xe-linux-distribution:138]: You text editor removed end of line whitespaces, but this does not belong to this very PR (could be a separate one).

Comment (mco-system) [reply mk/xe-linux-distribution]: @stormi trailing white spaces have been restored

**Comment by stormi** [comment]
Looks good to me but I'm not a repo owner. Could you squash your four commits together so that the PR contains only one, clean commit?

**Comment by mco-system** [comment]
@stormi Hoping this on is the good one :)

Sorry for the hard work. This is my first contrubution to some official repo. I'm not used to such rigor.

File: mk/testcases/slackware-15.0 [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+VERSION="15.0"
+ID=slackware
+VERSION_ID=15.0
+PRETTY_NAME="Slackware 15.0 x86_64 (post 15.0 -current)"
+ANSI_COLOR="0;34"
+CPE_NAME="cpe:/o:slackware:slackware_linux:15.0"
+HOME_URL="http://slackware.com/"
+SUPPORT_URL="http://www.linuxquestions.org/questions/slackware-14/"
+BUG_REPORT_URL="http://www.linuxquestions.org/questions/slackware-14/"
+VERSION_CODENAME=current
```
Comment (minli1) [inline mk/testcases/slackware-15.0:11]: please add a new line for this file.

**Review by minli1 — CHANGES_REQUESTED** [review]

File: mk/testcases/slackware-15.0 (hunk 2 above)
Comment (mco-system) [reply mk/testcases/slackware-15.0:11]: @minli1 The new line at the end of `/etc/os-release` is inexistant on the dozens of systems  I have access to (Slackware, Debian, RockyLinux, freeBSD, etc).

Do you really need it ?

Comment (minli1) [reply mk/testcases/slackware-15.0:11]: that's fine. we could add them if needed.

**Review by minli1 — APPROVED** [review]

**Review by xihuan-citrix — APPROVED** [review]

**Pull request merged**